### ISSUE TITLE
New Listings - remove pagesize constraint on past challenges

### ISSUE THREAD

**Issue by ajefts** [appirio-tech/topcoder-app#1120]
The past challenges being limited in the api call with pageSize=20.  This prevents the user from seeing more than 20 challenges, regardless of how they filter.

Remove the constraint.  If we need to set some limit for performance reasons, it should be something pretty high.

**Comment by ajefts** [comment]
Update:  This seems to getting applied when you select Past Challenges in the sidebar and then use the filter at the top.  For example, filter on Subtrack = Code after you select Past Challenges.

![image](https://cloud.githubusercontent.com/assets/3812270/24568812/69f794de-1631-11e7-99ea-b635f34ae7a9.png)

**Comment by ajefts** [comment]
I found at least part of the issue to be in ChallengeFiltersExample.jsx

it's making explicit calls to the past challenge apis with 20 set.  I bumped it up to 100 (the max allowed pagesize in the api for now, but this needs to be improved.  This should be calling the API based on the filter criteria.  Especially for past challenges, infinite scroll will be pointless if it doesn't refetch from the api using pageIndex and pageSize.
